workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -116,9 +107,40 @@ export const WorkflowInstancePage = ({
   );
   const [isAbortConfirmationDialogOpen, setIsAbortConfirmationDialogOpen] =
     useState(false);
-  const [isAbortAlertDialogOpen, setIsAbortAlertDialogOpen] = useState(false);
-  const [abortWorkflowInstanceErrorMsg, setAbortWorkflowInstanceErrorMsg] =
-    useState('');
+
+  const [isAborting, setIsAborting] = React.useState(false);
+  const [open, setOpen] = React.useState(false);
```

Comment:
rename the variable to isAbortSnackbarOpen
I couldn't figure out why it's needed